core/src/main/java/org/apache/metamodel/query/parser/SelectItemParser.java

```diff
@@ -157,6 +161,21 @@ public SelectItem findSelectItem(String expression) throws MultipleSelectItemsPa
             if ("*".equals(columnName)) {
                 throw new MultipleSelectItemsParsedException(fromItem);
             } else if (fromItem.getTable() != null) {
+                if (selectItemHasParenthesis) {
+                    if (functionName.equals(FunctionType.CONCAT.getFunctionName())) {
+                        String[] columnAndParametersAsString = expression.split(",");
+                        Object[] columnAndParameters = new Object[columnAndParametersAsString.length];
+                        int columnAndParamsIndex = 0;
+                        for (String parameter : columnAndParametersAsString) {
+                            Column column = fromItem.getTable().getColumnByName(parameter);
+                            if (column != null) columnAndParameters[columnAndParamsIndex] = column;
+                            else columnAndParameters[columnAndParamsIndex] = parameter.substring(1,parameter.length()-1);
```

Comment:
We should follow code conventions here and have newline and curly braces so that it becomes:
```
if (something) {
  something;
} else {
  something;
}
```